src/components/init-modal/index.js

```diff
@@ -24,12 +24,12 @@ export default ( { hasKeys, onSetupStatus } ) => {
 			shouldCloseOnClickOutside={ false }
 			shouldCloseOnEsc={ false }
 			title={
-				hasKeys
-					? __( 'Select a layout for your newsletter', 'newspack-newsletters' )
-					: __( 'Configure the newsletters', 'newspack-newsletters' )
+				shouldDisplaySettings
+					? __( 'Configure the newsletters', 'newspack-newsletters' )
```

Comment:
"Configure the newsletters" has bugs me a little. What about simply "Get started"? Or "Configure plugin"?